rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -42,94 +42,126 @@
 log = RotkehlchenLogsAdapter(logger)
 
 SMALLEST_AIRDROP_SIZE = 20900
-AIRDROPS = {
+AIRDROPS: dict[str, tuple[str, Asset, str, str, str]] = {
```

Comment:
Can you maybe change this tu use `NamedTuples`. It will help us with the typing and also make the code more redeable